articles/quickstart/spa/vuejs-beta/index.yml

```diff
@@ -0,0 +1,56 @@
+title: Vue
+# TODO remove 'image' once new QS page is live. Then only use 'logo'.
+image: /media/platforms/vue.png
+logo: vuejs
+beta: true
+language:
+  - Javascript
+author:
+  name: Frederik Prijck
+  email: frederik.prijck@auth0.com
+  community: false
+framework:
+  - Vue.js
+topics:
+  - quickstart
+contentType: tutorial
+useCase: quickstart
+show_releases: true
+sdk:
+  name: auth0-vue
+  url: https://www.github.com/auth0/auth0-vue
+  logo: vuejs
+snippets:
+  dependencies: application-platforms/vuejs/dependencies
+  setup: application-platforms/vuejs/setup
+  use: application-platforms/vuejs/use
+alias:
+  - vue
+  - vuejs
+  - vue.js
+seo_alias: vuejs
+default_article: 01-login
+articles:
+  - 01-login
+show_steps: true
+github:
+  org: auth0-samples
+  repo: auth0-vue-samples
+  branch: beta
+requirements:
+  - Vue 3+
```

Comment:
Do we need a minimum node version in here too?

Reply:
I think these would be defined by Vue, and I don't think we need to call it out explicitly.